xml/admin_troubleshooting.xml

```diff
@@ -0,0 +1,318 @@
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Troubleshooting</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker/>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    This chapter summarizes frequent problems that occur while using
+    &productname; and their solutions.
+   </para>
+  </abstract>
+ </info>
+
+ <sect1 xml:id="sec.admin.troubleshooting.overview">
+  <title>Overview</title>
+  <para>
+   This chapter is a collection of frequent problems that are reported
+   for &productname;. Additionally, &suse; support collects problems
+   and their solutions online at <link
+   xlink:href="https://www.suse.com/support/kb/?id=SUSE_CaaS_Platform"
+   />.
+  </para>
+  <para>
+   In case of problems, a detailed system report can be created with
+   the <command>supportconfig</command> command line tool. It will
+   collect information about the system such as: current kernel
+   version, hardware, installed packages, partition setup, and much
+   more. The result is a TAR archive of files. After opening a Service
+   Request (SR), you can upload the TAR archive to Global Technical
+   Support. It will help to locate the issue you reported and to assist
+   you in solving the problem. For details, see <link xlink:href=
+   "https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_adm_support.html"
+   />.
+  </para>
+  <para>
+   While resolving a technical difficulty with SUSE Support, you may
+   receive a so-called Program Temporary Fix (PTF). PTFs may be issued
+   for various packages as RPMs. For details about handling PTFs, see
+   <xref linkend="sec.admin.software.patch" />.
+  </para>
+  <para>
+   For details about how to contact &suse; support, refer to <link
+   xlink:href=
+   "https://www.suse.com/media/flyer/suse_customer_support_quick_reference_guide_flyer.pdf"
+   /> and <link xlink:href="https://www.suse.com/support/" />.
+  </para>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.troubleshooting.failed_bootstrap">
+  <title>Debugging Failed Bootstrap</title>
+  <para>
+   If the bootstrapping as described in <xref
+   linkend="sec.deploy.install.bootstrap" /> fails, there are several
+   places to look for errors. The following procedure outlines where to
+   start debugging.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Check the logs from Velum dashboard. Execute on the &admin_node;:
+    </para>
+    <screen>&prompt.root.admin;<command>docker logs $(docker ps | grep "velum-dashboard" | awk '{print $1}')</command></screen>
+   </step>
+   <step>
+    <para>
+     Check logs from Salt orchestration. If &productname; cluster
+     bootstrap fails in Velum, look into <command>salt-master</command>
+     orchestration logs by executing:
+    </para>
+<screen>&prompt.root.admin;<command>docker exec -it $(docker ps | grep "salt-master" | awk '{print $1}') \
+    salt-run state.event pretty=True > salt-event.logs</command></screen>
```

Comment:
This will give no information about past events. This will stream the current happening events as they come to the salt event bus. I'd remove this one from here, as this one is useful when an operation is ongoing at the moment (an orchestration is running), but not when something already failed.